From: Yaroslav Halchenko <debian@onerussian.com>
Subject: For backports to older suites/releases -- allow to have no cython3

achieved through removing versioning on cython itself at all and allowing older
cythons than when cython3 was introduced while requiring cython3

Origin: NeuroDebian
Last-Update: 2012-12-14

--- a/debian/control	2012-12-13 22:48:33.000000000 -0500
+++ b/debian/control	2012-12-13 22:54:11.000000000 -0500
@@ -6,7 +6,7 @@
 Build-Depends: debhelper (>= 7.0.50),
                python-all-dev (>= 2.5),
                python-setuptools,
-               cython (>= 0.17.1~),
+               cython,
                python-numpy (>= 1:1.6.1~), python-dateutil,
                python-scipy,
                python-tz,
